clowdapp.yaml

```diff
@@ -63,12 +63,12 @@ objects:
             value: ${DB_POOL_SIZE}
           - name: DB_MAX_OVERFLOW
             value: ${DB_MAX_OVERFLOW}
-    - name: processor
+    - name: inventory-events-processor
       replicas: ${{PROCESSOR_REPLICA_COUNT}}
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}
         command: ["bash"]
-        args: ["-c", "python -m ros.processor.main"]
+        args: ["-c", "python -m ros.processor.inventory_events_consumer"]
```

Comment:
With these changes, do you think we still need [ros/processor > main.py](https://github.com/RedHatInsights/ros-backend/blob/main/ros/processor/main.py) file?

Reply:
Yes, we need it for local setup.

Reply:
In case we get rid of the main.py (runner or application.py) file it might be difficult to implement app factory in future, in case we would like to.

Factory Pattern helps with,

1. Stop/avoid circular import;
2. Better and easier setup, for example, test, different environments;
3. More organized and readable project.

W.r.t No. 2 , one such use case for out project is the usage of `app` object in `test_api_endpoints.py`, it makes easier to test endpoints.

Reference: https://flask.palletsprojects.com/en/2.2.x/patterns/appfactories/

P.S. This pattern can be implemented in the `__init__.py`, but most of the blogs and official documentation suggests to have this in the `app.py/application.py`/runner.